src/components/20-molecules/datepicker/README.md

```diff
@@ -139,13 +139,21 @@ Boolean `invalid`, when set to true, forces showing the error message set with `
 
```

Comment:
The problem here is, that the input doesn't fills  up the space that the parent provides, even when `width="auto"`

Reply:
The datepicker does fill up 100%, but not the input-field itself.

Reply:
It makes no sense to stretch input-field always to 100%. So there is a max-length. I updated the readme.

Reply:
@danyball input field should stretch to 100% of its container (not 100% of vw), which is the same as all other inputs elements.